rotkehlchen/tests/unit/test_trades.py

```diff
@@ -137,14 +138,61 @@ def test_query_trade_history_online_but_exchange_excluded(events_historian):
     that exchange is not connected, but is also at the excluded exchanges we
     don't end up querying trades of all exchanges"""
 
-    patch_latest_trades = patch.object(events_historian, 'query_location_latest_trades')
-    patch_iterate_exchanges = patch.object(events_historian.exchange_manager, 'iterate_exchanges')
+    exchange = create_test_binance(
+        name='Binance',
+        database=events_historian.db,
+        msg_aggregator=events_historian.msg_aggregator,
+    )
```

Comment:
instead of this you can use the fixture `function_scope_binance`. Later instead of using `exchange` you use `function_scope_binance`